scripts/setupBuild.js

```diff
@@ -0,0 +1,10 @@
+const { writeFileSync, readFileSync } = require('fs');
+const { resolve } = require('path');
+
+const tsConfigPath = resolve(__dirname, '../tsconfig.json');
+const tsConfigRaw = readFileSync(tsConfigPath);
+const tsConfig = JSON.parse(tsConfigRaw);
+
+tsConfig.compilerOptions.sourceMap = true;
+
+writeFileSync(tsConfigPath, JSON.stringify(tsConfig, null, 2));
```

Comment:
I think better create new config and use it as argument for ts compiler

Reply:
We have a separate tsconfig for each package 😞 , creating seperate config for each is not practical. This script runs only in ci so shouldn't be a problem.